src/sentry/static/sentry/app/views/userFeedback/index.tsx

```diff
@@ -30,13 +29,10 @@ type State = AsyncView['state'] & {
 
 type Props = RouteComponentProps<{orgId: string}, {}> & {
   organization: Organization;
+  finishProfile: () => void;
```

Comment:
Is there a way to refer to `InjectedProps` via `withProfile`?

Reply:
There might be a way to infer it from the HoC, but I think the signature is simple enough to do this